### DISCUSSION TITLE
Yarn VS NPM

### DISCUSSION THREAD

**Discussion by JackTelford** [DustinBrett/daedalOS#466] [Q&A]
What type of Difficulty would it be to switch this project to a NPM build from Yarn do to Yarn not having a no--Save function for my package.json in C# Project.

Thank you!

**Comment by DustinBrett** [comment]
It probably wouldn't be too hard, but at the moment I am still happy with yarn for most of my use cases for the project. You could probably clone the repo and convert the `yarn.lock` file to work with npm instead. Did you run into issues doing this?
